http-server-jetty/build.gradle

```diff
@@ -2,6 +2,11 @@ plugins {
     id 'io.micronaut.build.internal.module'
 }
 
+repositories {
```

Comment:
Use a base module to reduce duplication. see https://github.com/micronaut-projects/micronaut-aws/pull/1445